pkg/kubelet/kubelet_pods.go

```diff
@@ -1653,14 +1655,15 @@ func (kl *Kubelet) convertStatusToAPIStatus(pod *v1.Pod, podStatus *kubecontaine
 		ecSpecs,
 		len(pod.Spec.InitContainers) > 0,
 		false,
+		true,
```

Comment:
why is it true for ephemeral containers? they are not started in the beginning, only started when requested